### DISCUSSION TITLE
Changing email address for billing

### DISCUSSION THREAD

**Discussion by jdbann** [plausible/analytics#767] [Anything Else]
Hey hi!

Is it possible to change the email address used for my billing? Updating my email address in account settings doesn't currently update the billing email address and I wasn't able to change it as part of the 'update billing info' flow with Paddle.

Thanks

**Comment by metmarkosaric** [comment]
thanks for reaching out @jdbann! we don't have an easy self-serve way to do that at this moment unfortunately but we do plan to add it. would you be able to [email us](https://plausible.io/contact) with the details and we'll get this manually sorted for you? thanks!

**Reply by jdbann** [reply]
Ah great! Tried to find a support email but feel like a fool for missing the 'Contact' page 😂 I've sent an email now. Much appreciated @metmarkosaric!

**Reply by metmarkosaric** [reply]
you're welcome!
